drivers/display/ssd1306.py

```diff
@@ -114,32 +117,44 @@ def write_data(self, buf):
     def poweron(self):
         pass
 
-# TODO convert this class to use the new hardware API
+
 class SSD1306_SPI(SSD1306):
     def __init__(self, height, spi, dc, res, cs=None, external_vcc=False):
         rate = 10 * 1024 * 1024
-        spi.init(spi.MASTER, baudrate=rate, polarity=0, phase=0)
-        dc.init(dc.OUT, dc.PULL_NONE, value=0)
-        res.init(res.OUT, dc.PULL_NONE, value=0)
+        spi.init(baudrate=rate, polarity=0, phase=0)
+        dc.init(dc.OUT, value=0)
+        res.init(res.OUT, value=0)
         if cs is not None:
-            cs.init(cs.OUT, cs.PULL_NONE, value=0)
+            cs.init(cs.OUT, value=0)
```

Comment:
The reason it was initially low is because in the original code the CS pin was not used past initialisation.  So the device was always "selected" and there was no need to toggle the CS pin around transfers.